lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -343,7 +343,7 @@
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
             <?php //@TODO @fenders look into adding a .-yelling class that all caps things ?>
-            <h3 class="inline-alt-text-color"><?php print t('JOIN OTHERS MAKING A DIFFERENCE'); ?></h3>
+            <h2 class="heading -emphasized"><?php print t('JOIN OTHERS MAKING A DIFFERENCE'); ?></h2>
```

Comment:
If not, it feels like this is perhaps a whole new style variant? Nothing we need to necessarily address in this PR, but worth thinking about if we see this all-caps no-visible-underline heading style appear elsewhere.